charts/mirrorbits/tests/custom_values_test.yaml

```
@@ -94,6 +134,9 @@ tests:
       - equal:
           path: spec.template.spec.containers[0].volumeMounts[2].mountPath
           value: /var/log/mirrorbits
+      - equal:
+          path: spec.template.spec.affinity.mirrorbits.podAntiAffinity.requiredDuringSchedulingIgnoredDuringExecution[0].topologyKey
```

```suggestion
          path: spec.template.spec.affinity.podAntiAffinity.requiredDuringSchedulingIgnoredDuringExecution[0].topologyKey
```

Otherwise the semantic is invalid (Kubernetes does not allow the `mirrorbits` keyword as child of `affinity`).